app/assets/javascripts/components/jquery_wrappers/InputTagit.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+class InputTagit extends React.Component {
+  componentDidMount() {
+    this.loadTagit();
+  }
+
+  loadTagit() {
+    $(this.input).tagit(
+      this.tagitProperties()
+    ).on('change', this.props.onChange);
+  }
+
+  tagitProperties() {
+    return {
+      availableTags: this.props.availableLabels,
+      readOnly: this.props.disabled,
+    };
+  }
+
+  render() {
+    return (
+      <input ref={ input => { this.input = input } } { ...this.props.input } tabIndex='this.props.labels' />
```

Comment:
Shouldn't the value of `tabIndex` be `{this.props.availableLabels}` (inside braces) instead the string `'this.props.labels'`?